lib/rambulance/exceptions_app.rb

```diff
@@ -59,12 +59,16 @@ def process_action(*)
         request.GET
       rescue ActionController::BadRequest
         request.env["MALFORMED_QUERY_STRING"], request.env["QUERY_STRING"] = request.env["QUERY_STRING"], ""
+      rescue Mime::Type::InvalidMimeType
+        request.env["MALFORMED_CONTENT_TYPE"], request.env["CONTENT_TYPE"] = request.env["CONTENT_TYPE"], "text/plain"
       end
 
       begin
         request.POST
       rescue ActionController::BadRequest
         request.env["MALFORMED_BODY"], request.env["rack.input"] = request.env["rack.input"], StringIO.new
+      rescue Mime::Type::InvalidMimeType
+        request.env["MALFORMED_CONTENT_TYPE"], request.env["CONTENT_TYPE"] = request.env["CONTENT_TYPE"], "text/plain"
```

Comment:
Let's remove this in favor of https://github.com/yuki24/rambulance/pull/55/files#r431494203.